osidb/models.py

```diff
@@ -940,6 +940,21 @@ def _validate_nist_rh_cvss_feedback_loop(self):
                 f"an explanation comment for RH CVSSv3 score.",
             )
 
+    def _validate_cvss_scores_and_nist_cvss_validation(self):
+        """
+        Checks that if nist_cvss_validation is set, then both NIST CVSSv3 and RH CVSSv3
+        scores need to be present.
+        """
+        if (
+            self.nist_cvss_validation
+            and self.cvss_scores.filter(version=CVSS.CVSSVersion.VERSION3).count() != 2
```

Comment:
This will work for now but if we ever introduce another issuer it will break. I am not sure we will ever do it but I would make this specific to NIST and RH anyway or at least make there a note that it must be updated if we ever have a third issuer.